File: PE-2/PE-2.js
var num1 = 2, num2 = 1, fibSum = 2;
while (num1 + num2 < 4000000) {
    var tempSum = num1 + num2;
    if (tempSum % 2 == 0) {
        fibsum += tempSum;
    }
    num2 = num1
    num1 = tempSum
}
console.log(fibSum)

File: PE-2/PE-2.py
num1, num2, fibSum= 2, 1, 2   # number 1 away, number 2 away, Fibonacci sum of evens after 1 and 2
while num2 + num1 < 4000000:
    tempSum = num1 + num2
    if tempSum % 2 == 0:
        fibSum+= tempSum
    num2 = num1
    num1 = tempSum

print(fibSum) # returns 4613732








    


